components/event/EventCard.vue

```diff
@@ -7,14 +7,16 @@ defineProps<{
 }>();
 
 const getPrice = (event: AnyEvent) => {
-  if (event.type === "workshop" && event.prices?.length > 0) {
-    const lowestPrice = event.prices.reduce(
-      (min, p) => (p.amount < min.amount ? p : min),
-      event.prices[0]
-    );
-    return `From ${lowestPrice.amount} ${lowestPrice.currency}`;
-  }
-  return event.price ? `${event.price.amount} ${event.price.currency}` : "Free";
+  if (!event.prices?.length) return "Free";
```

Comment:
if no prices defined, it doesn't mean it's free.
free is when `prices: [{amount:0}]`